config/environments/production.rb

```diff
@@ -3,18 +3,43 @@
   config.eager_load = true
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
+
+  # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
```

Comment:
We can just delete the comments here - I thought we might want them someday (e.g. `config.assets.css_compressor`), but happy to delete them (or uncomment them) too.

Reply:
@edwardloveall Does your 👍 mean "delete the comments in `config/environments/*.rb`"?

Reply:
It does! Sorry still haven't had a chance to look at the whole thing.